### ISSUE TITLE
Calibration settings date/time buttons overlap

### ISSUE THREAD

**Issue by MVAppleby** [peterdougstuart/PCWG#74]
The calibration settings buttons currently overlap on the GUI. Still possible to click both buttons, just 
difficult. Can only see right edge of the add date button.

![image](https://cloud.githubusercontent.com/assets/13198775/8614877/059be7ba-26e6-11e5-957b-7ad381a3d7b1.png)

**Comment by MVAppleby** [comment]
However if sectors or filters are open. The add date button comes to the top but the delete date button is completely covered.

![image](https://cloud.githubusercontent.com/assets/13198775/8615436/732786aa-26ea-11e5-9b5f-e06029647950.png)

**Comment by MVAppleby** [comment]
If general settings are opened the layout returns to normal:

![image](https://cloud.githubusercontent.com/assets/13198775/8615670/a62ce142-26ec-11e5-8338-802bdd52db45.png)

**Comment by MVAppleby** [comment]
I have created a fix for this, but it may affect the layout in ways i havent found yet.

I edited the datepickerentry code to move the layout of the clear and pick buttons using padx values.

https://github.com/MVAppleby/PCWG/commit/309954399d9b26c5053865bdf90be75df732b4f7#diff-a3803fe2234e186a0798f2820f406895L660

It now looks like this:

![image](https://cloud.githubusercontent.com/assets/13198775/8616416/1669cd3e-26f3-11e5-9637-5a5c9df1d78f.png)

and this:

![image](https://cloud.githubusercontent.com/assets/13198775/8616428/2480f406-26f3-11e5-8f87-642d7e8a419e.png)

I haven't merged this change with the master copy yet in case of possible bugs or if you would like the layout changed further.
